### ISSUE TITLE
Minikube Docker doesn't include docker-init

### ISSUE THREAD

**Issue by maiamcc** [kubernetes/minikube#5637]
Minikube's Docker doesn't seem to include `docker-init` binary.
```
$ mk version
minikube version: v1.4.0
commit: 7969c25a98a018b94ea87d949350f3271e9d64b6
```

**The exact command to reproduce the issue**:
```
$ minikube start
$ eval $(minikube docker-env)
$ docker pull alpine
$ docker run --init alpine
```
**The full output of the command that failed**:
```
$ docker run --init alpine
docker: Error response from daemon: exec: "docker-init": executable file not found in $PATH.
```

(In contrast, my system Docker runs the above command without error.)

**Docker Version**
```
# minikube Docker
$ docker version
Client: Docker Engine - Community
 Version:           19.03.1
 API version:       1.39 (downgraded from 1.40)
 Go version:        go1.12.5
 Git commit:        74b1e89
 Built:             Thu Jul 25 21:18:17 2019
 OS/Arch:           darwin/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.9
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.11.13
  Git commit:       039a7df9ba
  Built:            Wed Sep  4 16:55:50 2019
  OS/Arch:          linux/amd64
  Experimental:     false
```

contrast with my system Docker, which DOES include docker-init
```
# system Docker
$ docker version
Client: Docker Engine - Community
 Version:           19.03.1
 API version:       1.40
 Go version:        go1.12.5
 Git commit:        74b1e89
 Built:             Thu Jul 25 21:18:17 2019
 OS/Arch:           darwin/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.1
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.5
  Git commit:       74b1e89
  Built:            Thu Jul 25 21:17:52 2019
  OS/Arch:          linux/amd64
  Experimental:     true
 containerd:
  Version:          v1.2.6
  GitCommit:        894b81a4b802e4eb2a91d1ce216b8817763c29fb
 runc:
  Version:          1.0.0-rc8
  GitCommit:        425e105d5a03fabd737a126ad93d62a9eeede87f
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
```

**The output of the `minikube logs` command**: [logs here](https://gist.github.com/maiamcc/f37d35797ba7c44928034f4d074aecb3), though I doubt they contain anything useful

**The operating system version**: MacOS Mojave v. 10.14.6

**Comment by afbjorklund** [comment]
See https://github.com/kubernetes/minikube/pull/5607

**Comment by maiamcc** [comment]
Amazing, TY! I'm happy to close this issue/for you to close it whenever the PR hits a release, whatever works.

**Comment by afbjorklund** [comment]
It’s possible to add the missing binary from the docker tarball, if you need it before then. Or build/install `tini` and rename it. Will be fixed in 1.5+

**Comment by priyawadhwa** [comment]
Now that v1.5 has been released, docker-init should be available. I'm going to go ahead and close this issue.
